Sources/ManageNews.php

```diff
@@ -1008,7 +1008,7 @@ function SendMailing($clean_only = false)
 			$message = str_replace($from_member,
 				array(
 					$row['email_address'],
-					!empty($_POST['send_html']) ? '<a href="' . $scripturl . '?action=profile;u=' . $row['id_member'] . '">' . $cleanMemberName . '</a>' : ($context['send_pm'] ? '[url=' . $scripturl . '?action=profile;u=' . $row['id_member'] . ']' . $cleanMemberName . '[/url]' : $cleanMemberName),
+					!empty($_POST['send_html']) ? '<a href="' . $scripturl . '?action=profile;u=' . $row['id_member'] . '">' . $cleanMemberName . '</a>' : ($context['send_pm'] ? '[url=' . $scripturl . '?action=profile;u=' . $row['id_member'] . ']' . $cleanMemberName . '[/url]' : $scripturl . '?action=profile;u=' . $row['id_member']),
```

Comment:
As far as I can tell, the idea of `{$member.link}` seems to be to give a clickable version of the member's name that links to the member's profile. I figured that writing the name and then giving the URL to the profile was the best way to approximate that in a plain text email.

Maybe we should have both a `{$member.link}` variable and a `{$member.url}` variable? Then `{$member.link}` would give the clickable name in HTML emails and PMs or the nearest approximation we can provide in plain text emails. Meanwhile, `{$member.url}` would provide just the raw `$scripturl . '?action=profile;u=' . $row['id_member']` string so that admins could explicitly include that even in HTML emails whenever they chose.